source/core/Date.cpp

```diff
@@ -58,37 +55,42 @@ extern "C" {
 
 namespace Vireo {
 #if defined(VIREO_TYPE_Timestamp)
+
+    static const Int64 secondsInYear = 31536000;
+    static const Int64 secondsInLeapYear = 31622400;
+    static const Int32 secondsPerMinute = 60;
+    static const Int32 secondsPerHour = secondsPerMinute * 60;
```

Comment:
I think the convention to name the constants is prefix with 'k' (such as kTempFormattingBufferSize). I would rename the constants.